components/automate-chef-io/content/docs/saml.md

```diff
@@ -89,6 +89,12 @@ where:
 
 See the SAML Configuration Settings below for further configuration options.
 
+{{% warning %}}
+Azure AD lets you chose the _NameID_ field, and optionally apply _transformations_
+to it. The setting is only respected if the value selected in "Choose name identifier
```

Comment:
[suggestion only] I would be explicit about which setting (I'm assuming it's _transformations_).

Reply:
also if ur worried about being able to describe it correctly with words, we could put a screenshot here to help maybe?

Reply:
I think it's about every setting. I.e. if you configure the NameID field to be `user.petname`, and you don't change the name id format from "persistent" (to "email"), and A2 is configured to request name id policy format "email", azure ad's AuthnResponse will contain some sort of default for the "email" name id policy, presumably `user.email`.

Only if you _also_ change the drop down to "email" will A2 end up getting `user.petname` in the relevant NameID field.

However, the same applies when a transformation is applied (say, `ToLowercase(user.petname)`).

I have yet to verify this detail, however 🙄  -- would you have an idea how to change the text to make this more apparent?

Reply:
> also if ur worried about being able to describe it correctly with words, we could put a screenshot here to help maybe?

@susanev I'm happy to supply a screenshot, I'm just hesitant to blow this one detail out of proportion... what do you think?

Reply:
@srenatus Off the top of my head, the one idea I have is to state which things have to line up in order for stuff to work with A2, then say something like "That means that if you apply a transformation, stuff has to line up like X," and give a concrete example.